README.md

```diff
@@ -36,14 +38,69 @@ var Button = component({
 })
 
 // register as custom element
-Button = toCustomElement(customButton, 'button')
-document.registerElement('custom-button', Button)
+// The second parameter corresponds to a string Array containing the names of the attributes you'd like to observe and react to changes.
+var CustomButton = toCustomElement(Button, ['title'])
+document.registerElement('custom-button', CustomButton)
 
 // create new custom-button
-var button = document.createElement('button', 'custom-button')
+var button = document.createElement('custom-button')
 document.body.appendChild(button)
 ```
 
+## Custom Elements (webcomponents-v1)
+```js
+var toCustomElementV1 = require('nanocomponent-adapters/custom-element-v1')
+var component = require('nanocomponent')
+var html = require('bel')
+
+// create new nanocomponent
+var Button = component({
+  render: function (data) {
+    return html`
+      <button>hello planet</button>
+    `
+  }
+})
+
+// register as custom element
+// The second parameter corresponds to a string Array containing the names of the attributes you'd like to observe and react to changes.
+var CustomButton = toCustomElement(Button, ['title']);
+customElements.define('custom-button', CustomButton);
+// create new custom-button
+var button = document.createElement('custom-button')
+document.body.appendChild(button)
+```
+
+## Angular
+```js
```

Comment:
We can definitely show the es6 version of it. Let's keep in mind that the preferred way of developing Angular 2 applications it's using TypeScript and that for Custom Elements v2 you **need** to use the new class syntax for it to work. Prototypal inheritance will just not work.